README.md

```diff
@@ -1,2 +1,59 @@
-# ProjectReviewExample
-A dummy project to learn code review process by Microverse.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My YouTube Clone Project with Microverse
+
+> Description the project.
+
+## Built With
+
+- HTML
+- CSS
+- Git
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/egichuhis/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Have git installed on your local machine/PC.
+
+### Setup
+
+Open the terminal and run `git clone https://github.com/egichuhis/`
+
+### Usage
+
+Right-click on the index.html file and open with any browser.
+
+### Deployment
+
+You can push your project to github using `git add .` , `git commit -m "message"` and `git push`
+
+## Authors
+
+👤 **Edwin Gichuhi**
+
+- GitHub: [@githubhandle](https://github.com//)
+- Twitter: [@twitterhandle](https://twitter.com/)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in//)
```

Comment:
Please add valid links to your GitHub, LinkedIn, and Twitter. This will enable other users, recruiters, or any other parties to easily get in touch with you.